nested/src/syntax.js

```diff
@@ -19,6 +35,59 @@ const h_syntax_transforms = {
 		},
 	},
 
+	'text.html.basic': {
+		'$.variables': (h_vars) => {
+			h_vars.unquoted_attribute_value = /* syntax: sublime-syntax.regexp */ `(?:[^\\s<>/''"]|/(?!>))+`;
+		},
+
+		'$.contexts["style-type-attribute"][0].set': (a_rules) => {
+			a_rules.splice(1, 0, interp_escape_lookahead());
+		},
+
+		'$.contexts["script-type-attribute"][0].set': (a_rules) => {
+			a_rules.splice(1, 0, interp_escape_lookahead());
+		},
+
+		'$.contexts': (h_contexts) => {
+			// each context
+			for(let [si_context, a_rules] of Object.entries(h_contexts)) {
+				// an attribute equals context
+				if(si_context.endsWith('-attribute-equals')) {
+					a_rules.unshift(interp_escape_lookahead());
+				}
+			}
+		},
+
+		'$.contexts.main': (a_rules) => {
+			a_rules.splice(-2, 0, {
+				// Special case for handling ES tagged template where interpolation occurs in
+				// tag name position:
+
+				// <${expression} attr=val>
+
+				// If we did not include this case, the ‘<’ would be scoped as ordinary
+				// chardata.
+				match: /* syntax: sublime-syntax.regexp */ `(</?)(?=\\$\{)`,
```

Comment:
not important, but in case you would prefer not to have to worry about double escaping, you may like one of these alternatives:

```js
match: String.raw /* syntax: sublime-syntax.regexp */ `(</?)(?=\$\{)`,
match: /(<\/?)(?=\$\{)/.source,
```

Reply:
cool